lib/sprinkler_web/mqtt/handler.ex

```diff
@@ -15,10 +17,14 @@ defmodule SprinklerWeb.Mqtt.Handler do
     {:ok, state}
   end
 
-  def handle_message(["rs", _client_id, "telemetry"], _payload, state) do
+  def handle_message(["rs", client_id, "telemetry"], payload, state) do
     # What should we do with sensor information?
-    # IO.inspect(client_id)
-    # IO.inspect(payload)
+    decoded_payload = Jason.decode!(payload)
+
+    SprinklerWeb.Endpoint.broadcast(@telemetry_topic, "new_reading", %{
+      payload: decoded_payload,
+      device_id: client_id
```

Comment:
We need to convert this to an int since device ids will be ints as well when we fetch them from the database